guidelines/environments.md

```diff
@@ -1,38 +1,40 @@
 # Environments
 
-There are two pricing environments in which enterprises can operate - Gross or Net.
-* Gross Pricing Environments are environments in which taxes are *included* in the pricing that is offered to a customer, such as Germany, UK, Australia.
-* Net Pricing Environments are environments in which taxes are *excluded* in the pricing that is offered to a customer, such as in the US.
+Mews supports two main environments: Demo and Production. Demo can be used for integration development and testing.
+Production is for live customer sites.
 
-## Demo environments
+## Pricing environments
 
-The below addresses should be used for testing and development of the client applications.
+Enterprises can be configured for Gross Tax or Net Tax. In the Production environment this will depend on the individual enterprise.
+For development and test purposes, we have set up two versions of the Demo environment, one for Gross Tax and one for Net Tax.
 
-* **PlatformAddress** - `https://api.mews-demo.com`
-* **WebSocketAddress** - `wss://ws.mews-demo.com`
+* **Gross Pricing Environments**: These are environments in which taxes are *included* in the pricing that is offered to the end customer, such as used by Germany, UK and Australia.
+* **Net Pricing Environments**: These are environments in which taxes are *excluded* from the pricing that is offered to the end customer, such as used in the USA.
 
-Use the below logins for accessing the demo enterprises ([Gross Pricing Environment](#gross-pricing-environment) and [Net Pricing Environment](#net-pricing-environment)).
-
-* **MewsWebApplicationAddress** - `https://app.mews-demo.com`
-* **Email** - connector-api-demo@mews.com
-* **Password** - connector-API-2024
+## Demo environments
 
 ### Security Policy
 
-> **IMPORTANT!** The demo environments listed below are completely public and NO REAL DATA should be used for any reason. Failure to comply with these guidelines can result in immediate suspension of the connection or denial of certification.
+> **IMPORTANT!** The demo environments are completely public and NO REAL DATA should be used for any reason. Failure to comply with these guidelines can result in immediate suspension of the connection or denial of certification.
 
-### Request limits
+### Platform addresses
 
-* 500 requests per `AccessToken` within 15 minutes
-* 250 requests per endpoint per `AccessToken` within 15 minutes
+These addresses should be used for testing and development of client applications:
+
+* **PlatformAddress** - `https://api.mews-demo.com`
+* **WebSocketAddress** - `wss://ws.mews-demo.com`
+
+### Mews system credentials
 
-> **NOTE** All requests utilizing [Pagination](guidelines/pagination.md) have their limits increased by 10 times.
+These login credentials can be used to access __Mews Operations__ (for both Gross and Net Pricing Environments):
 
-### Gross Pricing Environment
+* **MewsWebApplicationAddress** - `https://app.mews-demo.com`
```

Comment:
nitpick: `MewsWebApplicationAddress` looks like a variable for parameters but it's not, maybe it could be just "Mews web application address"?